manager/dispatcher/dispatcher_test.go

```diff
@@ -48,6 +49,7 @@ func TestMessageHandler(t *testing.T) {
 		Msg:          "ExecuteMsg",
 		Severity:     pb.SEVERITY_WARNING,
 		ResourceType: "SamplePlugin",
+		Option:       pUnique0,
```

Comment:
How do you think using another field name like `ID` instead of `Option`?
The name `Option` seems like "Optional" but the unique value seems to be mandatory.

Reply:
> How do you think using another field name like `ID` instead of `Option`? The name `Option` seems like "Optional" but the unique value seems to be mandatory.

I used key name `Option` because to contain any value in it if necessary.

Reply:
As I understood unique value is added to distinguish from other messages.
Is it required just only for special cases?

Reply:
unique value is added to distinguish to track change of status and severity. 
I first created Options as type of string, but if there's any chance we need to contains any addition value requires in ReqMsg, it prefer to contain key options.

Reply:
Got it.